docs/pip.md

```diff
@@ -266,6 +266,42 @@ If you do need to use a git url, Cachi2 requires that it specifies a full commit
 git urls. Cachi2 requires hashes for https. Using one --hash makes pip require hashes for everything. Pip does not
 support hashes for git dependencies. Please use https urls instead.*
 
+### Supported options
+
+Requirements files support some `pip install` options - see
+<https://pip.pypa.io/en/stable/reference/requirements-file-format/#supported-options>.
+
+Cachi2 supports a small subset of them, ignores those that are not relevant for prefetching, and raises an error for
+those that *are* relevant but aren't supported.
+
+#### Global
+
+##### [`--index-url`](https://pip.pypa.io/en/stable/cli/pip_install/#install-index-url)
```

Comment:
Oh, one more thing (I guess too late for the older flags we support), we should start documenting the version of cachi2 where a feature or support for a particular domain was added, e.g. like here: https://github.com/eskultety/cachi2/blob/tmp/docs/gomod.md#go-121-since-cachi2-v050

Reply:
:+1: added (in the body, didn't want it to become part of the anchor for the heading https://github.com/chmeliik/cachi2/blob/document-index-url/docs/pip.md#--index-url)

Reply:
I'm fine with that, I just wanted to give an example, text is alright (probably a good future idea would be to unify these in style, i.e. make them bold/italic without creating an anchor or at least a hidden one).